e2e/issues_test.go

```diff
@@ -0,0 +1,81 @@
+// +build e2e
+
+//
```

Comment:
Can this be cleaned up or modified?

Can the filename be renamed to better capture at a high level what it tests? Something like `render_test.go` -- `issues_test.go` reads to me more related to GitHub than the content of the test itself.

Reply:
I wasn't 100% on the name but wanted to call out that this was a bug related test, targeting a specific issue, and not a general/feature/golden-path e2e test. I'd be happy to change the name and use a comment to call out the issue, but I thought maybe a bit more info (filename, common subdir) might be useful.

Reply:
Gotcha, yeah I saw the comment on [#L21](https://github.com/hashicorp/consul-terraform-sync/pull/244/files/c7c65a7ad81f0d11fa88f6cc28c7b670245b2911..13c493ef5855cdf7830a8738fe6eab80152a1a2c#diff-986d4a3319e0f499de5e5970cb965cde1c2571d7d9502c113c082b0321bb8e8cR21).

If you imagine this test file persisting for a while, I'd prefer it to be renamed since there are many github issues and we'll likely write more tests targeting bugs captured in gh issues.

Reply:
Cool. I'll rename it. :+1:

Reply:
Oh... and yeah, I'll get rid of that empty comment.